src/Migrations/RevitNodes/SunPath.cs

```diff
@@ -1,91 +0,0 @@
-﻿using System;
-using System.Linq;
-using System.Xml;
-using Dynamo.Graph;
-using Dynamo.Graph.Nodes;
-using Dynamo.Models;
-using Dynamo.Migration;
-
-namespace Dynamo.Nodes
-{
-    public class SunPathDirection : MigrationNode
-    {
-        [NodeMigration(from: "0.6.3.0", to: "0.7.0.0")]
-        public static NodeMigrationData Migrate_0630_to_0700(NodeMigrationData data)
-        {
-            var migrationData = new NodeMigrationData(data.Document);
-            XmlElement oldNode = data.MigratedNodes.ElementAt(0);
-
-            // Create nodes
-
-            XmlElement sunPathNode = MigrationManager.CloneAndChangeName(
-                oldNode, "DSRevitNodesUI.SunPathDirection", "SunPath Direction");
-            sunPathNode.SetAttribute("guid", Guid.NewGuid().ToString());
-            sunPathNode.SetAttribute("x", (Convert.ToDouble(oldNode.GetAttribute("x")) - 230).ToString());
-
-            var vectorAsPoint = MigrationManager.CreateFunctionNodeFrom(oldNode);
-            MigrationManager.SetFunctionSignature(vectorAsPoint, "ProtoGeometry.dll",
-                "Vector.AsPoint", "Vector.AsPoint");
-            
-            migrationData.AppendNode(sunPathNode);
-            migrationData.AppendNode(vectorAsPoint);
-
-            // Update connectors
-            migrationData.CreateConnector(sunPathNode, 0, vectorAsPoint, 0);
-
-            return migrationData;
-        }
-
-        [NodeMigration(from: "0.7.0.0", to: "0.7.3.0")]
-        public static NodeMigrationData Migrate_0700_to_0730(NodeMigrationData data)
```

Comment:
This node migration is for 0.7.0 to 0.7.3, what happens to 0.7.1 node, will that also execute this code?